typescript/infra/config/environments/mainnet2/chains.ts

```diff
@@ -12,12 +12,12 @@ export const mainnetConfigs: ChainMap<ChainMetadata> = {
     ...chainMetadata.polygon,
     blocks: {
       ...chainMetadata.polygon.blocks,
-      confirmations: 3,
+      confirmations: 10,
     },
     transactionOverrides: {
-      maxFeePerGas: 500 * 10 ** 9, // 500 gwei
-      maxPriorityFeePerGas: 100 * 10 ** 9, // 100 gwei
-      // gasPrice: 50 * 10 ** 9, // 50 gwei
+      //maxFeePerGas: 500 * 10 ** 9, // 500 gwei
+      //maxPriorityFeePerGas: 100 * 10 ** 9, // 100 gwei
+      gasPrice: 250 * 10 ** 9, // 250 gwei
```

Comment:
Just checking, can you confirm this switch back to old gas tx types was intentional?